changelogs/CHANGELOG-v0.4.0.md

```diff
@@ -0,0 +1,134 @@
+
+The v0.4.0 release is a minor release which fully transitions bpfman to be a
+library rather than a daemon.  From a user's perspective the main
+difference is that the `bpfctl` CLI can now simply be called directly with the
+`bpfman` binary, otherwise all of the major commands remain the same.
+
+On top of transitioning to a library the community has also implemented some
+exciting new features:
+
+- Support for Uprobe Programs
+- The ability to attach Uprobes inside containers (locally and in Kubernetes), see [the blog](https://bpfman.io/main/blog/2024/02/26/technical-challenges-for-attaching-ebpf-programs-in-containers/) for more.
+- Support for Kprobe Programs
+- Support for Fentry Programs
+- Support for Fexit Programs
+
+Additionally this release provides some new binary crates, the `bpf-metrics-exporter`,
+and the `bpf-log-exporter`, which allow users to gather useful information regarding
+the bpf subsystem.
```

Comment:
Do we need to mention the bpfman-rpc binary (it's new for this release)?

Reply:
Yeah I can give it a call out!